engine/entity/root/window.h

```diff
@@ -8,8 +8,6 @@
 
 namespace chira {
     class Window : public Frame {
-        // We don't want people making windows and adding them to the entity tree
-        // Only the Engine can make windows!
```

Comment:
I would have liked to keep it this way, isn't it possible to make a templated function to do this in a window manager class?